src/app/map/page.tsx

```
@@ -1,224 +1,223 @@
+import { useEffect, useState, Suspense, useRef } from 'react';
 import { useSearchParams } from 'next/navigation';
 import dynamic from 'next/dynamic';
-import { getBoardPins, getProgress, getProgressCountdown, getVoteVenuePins, getAreaList } from '@/lib/api';
-import { getStatusText, getStatusColor, createProgressBox, createProgressBoxCountdown, createBaseLayers, createGrayIcon } from '@/lib/map-utils';
-import { PinData, VoteVenue, AreaList } from '@/lib/types';
+import type { Map as LeafletMap, LayerGroup, Control } from 'leaflet';
+import { getBoardPins, updatePin } from '@/lib/api';
+import { getStatusText, getStatusColor, createBaseLayers } from '@/lib/map-utils';
+import { PinData } from '@/lib/types';
 
 const Map = dynamic(() => import('@/components/Map'), { ssr: false });
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)